<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="37336" started-at="2015-07-21T04:36:34Z" finished-at="2015-07-21T04:37:11Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="37336" started-at="2015-07-21T04:36:34Z" finished-at="2015-07-21T04:37:11Z">
      <class name="com.mycompany.testUsingMaven.TestMcAfeeLoginPage">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.mycompany.testUsingMaven.TestMcAfeeLoginPage@1d251891]" name="setup" is-config="true" duration-ms="2433" started-at="2015-07-20T21:36:34Z" finished-at="2015-07-20T21:36:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="testMcAfeeLoginPageTitle()[pri:0, instance:com.mycompany.testUsingMaven.TestMcAfeeLoginPage@1d251891]" name="testMcAfeeLoginPageTitle" duration-ms="34315" started-at="2015-07-20T21:36:37Z" finished-at="2015-07-20T21:37:11Z">
          <exception class="org.openqa.selenium.StaleElementReferenceException">
            <message>
              <![CDATA[Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 32.29 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.46.0', revision: '61506a4624b13675f24581e453592342b7485d71', time: '2015-06-04 10:22:50'
System info: host: 'wqa-test-PC', ip: '10.0.0.23', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_51'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=28.0, platform=WINDOWS, browserConnectionEnabled=true, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 1f98e011-be61-4a9b-868b-0d5fe23d94e5]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 32.29 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.46.0', revision: '61506a4624b13675f24581e453592342b7485d71', time: '2015-06-04 10:22:50'
System info: host: 'wqa-test-PC', ip: '10.0.0.23', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_51'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=28.0, platform=WINDOWS, browserConnectionEnabled=true, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 1f98e011-be61-4a9b-868b-0d5fe23d94e5
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:605)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:269)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:80)
	at com.mycompany.testUsingMaven.TestMcAfeeLoginPage.testMcAfeeLoginPageTitle(TestMcAfeeLoginPage.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.46.0', revision: '61506a4624b13675f24581e453592342b7485d71', time: '2015-06-04 10:22:50'
System info: host: 'wqa-test-PC', ip: '10.0.0.23', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_51'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web-element-cache.js:8981)
	at <anonymous class>.Utils.getElementAt(file:///C:/Users/wqa-test/AppData/Local/Temp/anonymous1064949472510937627webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:8574)
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/wqa-test/AppData/Local/Temp/anonymous1064949472510937627webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:9613)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/wqa-test/AppData/Local/Temp/anonymous1064949472510937627webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12265)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/wqa-test/AppData/Local/Temp/anonymous1064949472510937627webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12282)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/wqa-test/AppData/Local/Temp/anonymous1064949472510937627webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:603)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.StaleElementReferenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMcAfeeLoginPageTitle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.mycompany.testUsingMaven.TestMcAfeeLoginPage@1d251891]" name="tearDown" is-config="true" duration-ms="557" started-at="2015-07-20T21:37:11Z" finished-at="2015-07-20T21:37:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.mycompany.testUsingMaven.TestMcAfeeLoginPage -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
